Symfony/CS/Fixer/PSR0/Psr0Fixer.php

```diff
@@ -161,6 +160,12 @@ public function supports(\SplFileInfo $file)
             return false;
         }
 
+        $tokens = token_get_all(sprintf('<?php %s', $filenameParts[0]));
```

Comment:
hmm afraid not so, it provides the `[int] type/kind - [int] value`, but I need to check against the string as used in the code itself.